docs/src/appendix/migrating-from-chiseltest.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Migrating from ChiselTest"
+sidebar_position: 0
+---
+
+# Migrating from ChiselTest to ChiselSim
+
+## Background
+
+With the release of Chisel 5, Chisel moved off of the legacy [Scala FIRRTL Compiler](https://github.com/chipsalliance/firrtl) to the MLIR FIRRTL Compiler (part of the [llvm/circt](https://github.com/llvm/circt) project).
+This change in underlying compiler technology has been a crucial piece in enabling the addition of many new features to Chisel, including linear-temporal logic (LTL) properties, Probes, and Layers.
+Unfortunately, Chisel 3's testing library, [ChiselTest](https://github.com/ucb-bar/chiseltest), is built around the SFC, making it difficult to support ChiselTest in Chisel 5 and beyond.
+
+This is not a fundamental problem, it is one of engineering resources.
```

Comment:
I would just strike this and pivot to talking about ChiselSim:

```suggestion
ChiselSim is the approved replacement for ChiselTest in Chisel 5 and beyond.
ChiselSim is maintained and used by the core Chisel development team.
```

Reply:
I also was looking for a change to this line, and I like this a lot 👍